js/dimensions.js

```diff
@@ -1,4 +1,15 @@
 $(document).ready(function() {
+  /*
+  These limits are needed to prevent users from using large images to crop really small images.
+  The ORIGINAL constants mark the largest images that can be used to crop images less than or
+  equal to the dimensions of the CROPPED constants.
+  See this issue for more details: https://github.com/chapmanu/dimensions/issues/29
+  */
+  const ORIGINAL_SMALL_IMAGE_WIDTH_LIMIT = 800;
+  const ORIGINAL_SMALL_IMAGE_HEIGHT_LIMIT = 800;
```

Comment:
Where did this 800px limit come from? Was that based on your tests? It seems kinda small.

Did you product owner ok this?

Reply:
I went over the samples of cropped images with Ross. I showed him images the same images that are in the issue related to this story and he decided on 800px. To be honest, the results are harder to see in the issue than when I showed him straight off of dimensions.